main.js

```diff
@@ -318,6 +352,20 @@ class ModuleInstance extends InstanceBase {
 		// this.oscSocket.socket.on('ready', () => { })
 	}
 
+	/**
+	 * Reset our internal variables
+	 */
+	emptyEncVariables() {
+		let variableDefinitions = GetVariableDefinitions()
+		let updateDefs = {}
+		variableDefinitions.forEach( function( varDef ) {
+			if ( /^enc_/.test( varDef['variableId'] ) ) {
```

Comment:
```suggestion
			if (varDef['variableId'].startWith('enc_')) {
```
should be less expensive than a regex

Reply:
Broken, I think, should be .startsWith not .startWith.  I'll include that in my next pull request. I agree, though, better than regex. This broke the refreshing of variables upon changing of active fixture.